src/YesSql.Core/Sql/BaseComandInterpreter.cs

```diff
@@ -203,9 +203,7 @@ public virtual void Run(StringBuilder builder, IAddIndexCommand command)
 
         public virtual void Run(StringBuilder builder, IDropIndexCommand command)
         {
-            builder.AppendFormat("drop index {0} ON {1}",
-                _dialect.QuoteForColumnName(command.IndexName),
-                _dialect.QuoteForTableName(command.Name));
+            builder.Append(_dialect.GetDropIndexString(command.IndexName, command.Name));
```

Comment:
Ideally we could have created a method name `DropIndex` that would have accepted a `StringBuilder` (like `Page()`) and reuse this one in it. But that's ok, we don't drop indexes on all calls.

Reply:
Ah, ok. To reduce allocations of StringBuilder I assume, makes sense :)